text.go

```diff
@@ -30,12 +30,33 @@ const (
 	TextWrapWord
 )
 
+const (
+	// DefaultTabWidth is the default width in spaces
+	DefaultTabWidth = 4
+)
+
 // TextStyle represents the styles that can be applied to a text canvas object
 // or text based widget.
 type TextStyle struct {
 	Bold      bool // Should text be bold
 	Italic    bool // Should text be italic
 	Monospace bool // Use the system monospace font instead of regular
+	tabWidth  int  // Width of tabs in spaces
```

Comment:
Can't we just used a public field like the other items? the check for 0 can be done inside the text code instead?

Reply:
done

Reply:
I don't think this is done. The default code below is what causes the exposed public API above - if we just use a public field as suggested then the default handling would be in painter / widget and would avoid the additional API requirements.